client/src/hooks/graphql/use-with-data.tsx

```diff
@@ -88,12 +88,16 @@ export function useWithData<T>(fetch: () => Promise<T>): UseData<T> {
     try {
       await update.action(editedData);
     } catch (err) {
-      console.error(err);
-      dispatch({
-        type: LoadingActionType.SAVING_FAILED,
-        payload: { message: "Failed to save", error: err.message },
-      });
-      return;
+      if (err instanceof Error) {
+        console.error(err);
+        dispatch({
+          type: LoadingActionType.SAVING_FAILED,
+          payload: { message: "Failed to save", error: err.message },
+        });
+        return;
+      } else {
+        console.log("Unexpected error", err);
```

Comment:
You fixed the issue here in your other branch, so either copy your fixes here or wait for other branch to merge